### PR TITLE
Update project 1 README/Update project 2 to work with new versions of Terraform

### PR DESCRIPTION
**Pull request by cbuto**

### CODE REVIEW

File: project2/terraform/versions.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.13.2"
```
Comment (jadams-contsec) [inline project2/terraform/versions.tf:2]: Should this be `~> 0.13.2` instead, in order to keep it within the 0.13 family?

**Review by timwisbauer-contsec — APPROVED** [review]

File: project2/terraform/versions.tf (hunk 1 above)
Comment (kband) [reply project2/terraform/versions.tf:2]: Should we disallow older versions?

I think candidates will fall into two types:
* never terraformed before in their life, but going to give it a shot
* seasoned pros who will just make it work

Setting it to 1.0.x will remove one tiny edge case for the newbies, and not affect the pros at all.

Comment (cbuto) [reply project2/terraform/versions.tf:2]: I can definitely do that, I tested this against 1.X and it worked. But keeping it 0.13 would likely be best for now

Comment (cbuto) [reply project2/terraform/versions.tf:2]: i did test this on 1.0.6, so we can lock it down to 1.x if we want.

**Review by jadams-contsec — APPROVED** [review]

File: project2/terraform/versions.tf (hunk 1 above)
Comment (cbuto) [reply project2/terraform/versions.tf:2]: leaving this open for now so its more flexible for candidates and us

**Pull request merged**